designer/server/src/lib/fetch.js

```diff
@@ -9,6 +9,10 @@ import Wreck from '@hapi/wreck'
 export async function request(method, url, options) {
   const response = await Wreck.request(method, url.href, options)
 
+  if (response.statusCode === 404) {
+    return { response, body: undefined }
+  }
```

Comment:
This prevents us throwing errors and having to handle them internally

When requests to **forms-manager** throw an Error 404 we just return an empty body

(Similar to [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) where a 404 doesn't throw)